src/components/Form.js

```diff
@@ -0,0 +1,52 @@
+import { useState, useRef } from 'react';
+import styled from 'styled-components';
+import { v4 as uuidv4 } from 'uuid';
+
+export default function Form({ savedList, setSavedList }) {
+  const [input, setInput] = useState([]);
+  const onChange = (e) => {
+    setInput(e.target.value);
+  };
+
+  const nextId = useRef(1);
+
+  const onCreate = (e) => {
+    e.preventDefault();
+    const toDo = { id: uuidv4(), content: input, isDone: false };
+    const updatedList = [...savedList, toDo];
+    setSavedList(updatedList);
+    setInput('');
+    nextId.current += 1;
+  };
```

Comment:
새로운 투두를 입력할 때 중복되는 `id` 값이 없도록 작성하신 코드 같은데, 이미 `uuidv4()`를 사용하여 고유값을 부여했으므로 이 부분은 필요없어 보이네요

```suggestion


  const onCreate = (e) => {
    e.preventDefault();
    const toDo = { id: uuidv4(), content: input, isDone: false };
    const updatedList = [...savedList, toDo];
    setSavedList(updatedList);
    setInput('');

  };
```